### DISCUSSION TITLE
TSC Meeting, May 2020

### DISCUSSION THREAD

**Discussion by dbaluta** [thesofproject/sof#7384] [Team Posts]
Date:
Doodle: https://doodle.com/poll/u48rdqq94vs8gb74
Hangout URL: https://meet.google.com/vkq-bvnm-rkj
Participants:

### General

Updates from our latest TSC in February

### Firmware
1. SOF development model (imx-stable-v1.5 out) (Daniel)
2. Post process component (Daniel, Marcin)
3. Zephyr status (Liam)
4. IPC2.0 status (Liam)
5. Topology 2.0 status (Liam)
6. Testbench plans (Liam)
7. ABI process (Kai, Curtis)

### Driver

1. Compressed interface (Daniel, Marcin)

### Misc

1. GSoC & Outreachy updates (Daniel, Curtis, Ranjani)

**Comment by cujomalainey** [comment]
Fyi, the 25th is a holiday in NA

**Comment by cujomalainey** [comment]
Is the ABI versioning discussion still open? should we be talking about it here?

**Comment by lgirdwood** [comment]
@cujomalainey added, @kv2019i has been finalising this recently.
